data.json

```diff
@@ -38,6 +38,27 @@
       }
     }
   },
+  "ByteDance/Tiktok": {
+    "url": "https://job.toutiao.com/s/J7LBfB5",
+    "jobs": {
+      "softwareEngineer": {
+        "intern (referral code: VC7FSRM)": {
```

Comment:
`intern` and `newGrad` are special keys. If the keys don't match the data won't show up at all.